examples/blog-with-comment/components/comment/form.js

```diff
@@ -0,0 +1,45 @@
+import { useAuth0 } from '@auth0/auth0-react'
+
+function CommentForm({ text, textSet, onSubmit }) {
+  const { isAuthenticated, logout, loginWithPopup } = useAuth0()
+
+  return (
+    <form onSubmit={onSubmit}>
+      <textarea
+        className="flex w-full max-h-40 p-3 rounded resize-y bg-gray-200 text-gray-900 placeholder-gray-500"
+        rows="2"
+        placeholder={
+          isAuthenticated
+            ? `What are your thoughts?`
+            : 'Please login to leave a comment'
+        }
+        onChange={(e) => textSet(e.target.value)}
```

Comment:
setText maybe? So it follows what React Hooks normally do 😄

Reply:
fixed 😇